### PR TITLE
Instruments.xml

### PR DESCRIPTION
**Pull request by duck57**
Improve percussion instrument ranges (#111331)
Add single-staff marimba (#111431)
Add contrabass marimba

### CODE REVIEW

**Comment by duck57** [comment]
This is actually a duplicate of #2612 and should be ignored.

File: share/instruments/instruments.xml [hunk 1]
```diff
@@ -4760,16 +4760,43 @@
             <Instrument id="marimba">
                   <longName>Marimba</longName>
                   <shortName>Mrm.</shortName>
-                  <description>Marimba</description>
+                  <description>Marimba (grand staff)</description>
+<<<<<<< HEAD
+                  <trackName>Marimba (Grand Staff)</trackName>
+=======
+                  <trackName>Marimba (grand staff)</trackName>
+>>>>>>> origin/instruments.xml
```
Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml]: merge conflicts!

File: share/instruments/instruments.xml [hunk 2]
```diff
@@ -4760,16 +4760,43 @@
...
+            </Instrument>
+            <Instrument id="marimba-2">
+                  <longName>Marimba</longName>
+                  <shortName>Mrm.</shortName>
+                  <description>Single-staff marimba</description>
+<<<<<<< HEAD
+                  <trackName>Marimba (Single Staff)</trackName>
+=======
+                  <trackName>Marimba (single staff)</trackName>
+>>>>>>> origin/instruments.xml
```
Comment (Jojo-Schmitz) [inline share/instruments/instruments.xml]: merge conflicts!

**Comment by Jojo-Schmitz** [comment]
The merge commit shouldn't be in there, the conflicts in instruments.xml need to get resolved and  the chanmges in share/instruments/instrumentsxml.h are way do large.
Other than that you gave me the go at a PR, so I did #2612

**Comment by Jojo-Schmitz** [comment]
For the next PR: Never merge, but rebase instead, I use `git pull --rebase upstream master` for this.

**Comment by duck57** [comment]
I saw that there wasn't yet a PR last nights, so I figured I'd submit mine before bed (which was the same time you were submitting yours)

**Comment by Jojo-Schmitz** [comment]
And I did mine 1st thing in the morning ;-)

**Comment by Jojo-Schmitz** [comment]
I had some other changes to these files pending anyway

**Pull request closed without merging**